### ISSUE TITLE
Unable to click and move cursor in Redactor field

### ISSUE THREAD

**Issue by dyerc** [verbb/vizy#87]
**Description**
I've just come across this problem in Firefox (it works fine in Chrome). When left clicking somewhere in a Redactor field the cursor doesn't move anywhere. In the HTML view the same problem is present. If I right-click, the cursor moves correctly.

The Redactor field can still be used with the keyboard arrow keys.

**Steps to reproduce**
1. Set up a Vizy field with a Redactor text editor inside
2. Try clicking to navigate with the text cursor

**Additional info**
- Plugin version: 1.0.8
- Craft version: 3.7.18.2
- Multi-site: No

**Comment by ax2000** [comment]
Same here. 

Firefox seems to behave a bit odd with it with the above and also when applying styles to the selected text requires a few tries and I need to select out of the block and then back in in order to get the styles applied.

I've got same versions as @dyerc

**Comment by engram-design** [comment]
A known issue with Firefox https://github.com/verbb/vizy/issues/58 and trying to get it resolved in Tiptap https://github.com/ueberdosis/tiptap/issues/1668 - not just related to Redactor

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.9](https://github.com/verbb/vizy/releases/tag/1.0.9)
